.github/workflows/codeql-analysis.yml

```diff
@@ -18,18 +25,36 @@ jobs:
       security-events: write
 
     strategy:
-      fail-fast: false
       matrix:
         language: ['javascript']
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+        with:
+          # Fetch history for CodeQL analysis
+          fetch-depth: 0
+
+      # setup dependencies
+      - name: Setup Libssl
```

Comment:
This action has been deleted. I think it was not needed to run it here anyway.

Reply:
ok deleted